_posts/2020-02-22-gsoc2020.md

```diff
@@ -0,0 +1,115 @@
+---
+layout: post
+title: Google Summer of Code 2020
+---
+
+<p>
+<img
+src="https://developers.google.com/open-source/gsoc/images/gsoc2016-sun-373x373.png"
+title="Google Summer of Code 2020" alt="Google Summer of Code 2020"
+style="float: left; height: 4.5em; " />
+</p>
+
+MDAnalysis has been accepted as an [organization][org] into [Google Summer of Code 2020][gsoc]!
+If you are a student who is interested in working with us
+this summer, please read the advice and links below and write to us on the
+[mailing list].
+
+We are looking forward to all applications from interested students
+(*undergraduate* and *postgraduate*).
+
+The application window **deadline** is **March 31, 2020 at 18:00 (UTC)**. As
+part of the application process you must familiarize yourself with [Google
+Summer of Code 2020][gsoc]. 
+
+If you are interested in working with us please read on and contact us on our [mailing list].
+[Apply] as soon as possible; the application window opens on March 16, 2020.
+
+# Project Ideas
+
+If you have your own idea about a potential project we'd love to work
+with you to develop this idea; please write to us on the [developer
+list][mailing list] to discuss it there.
+
+We also have listed several [possible projects][ideas] for you to work
+on on our wiki. Our initial list of ideas (see summaries in the
```

Comment:
remove one "on"

Reply:
I think this is meant to be "work on, on our wiki"